ansible/roles/apps/stubbornjava/tasks/main.yml

```diff
@@ -0,0 +1,11 @@
+---
+  - name: Creating App Directory
+    file:
+      path: apps/stubbornjava
```

Comment:
I wanted to do `/apps/{appName}` but it needs better permissions. We can set up users per service and reorganize later.